packages/commons/src/config/kafkaConfig.ts

```diff
@@ -6,11 +6,13 @@ export const KafkaConfig = z
     KAFKA_CLIENT_ID: z.string(),
     KAFKA_GROUP_ID: z.string(),
     KAFKA_DISABLE_AWS_IAM_AUTH: z.literal("true").optional(),
+    KAFKA_TOPICS: z.string().transform((v) => v.split(",")), // comma-separated
```

Comment:
@MalpenZibo @Viktor-K FYI-2: we moved the the topics string value into the `kafgaConfig`